files/es/web/javascript/eventloop/index.html

```diff
@@ -18,7 +18,7 @@ <h2 id="Conceptos_de_un_programa_en_ejecución">Conceptos de un programa en eje
 
 <h3 id="Representación_visual">Representación visual</h3>
 
-<p style="text-align: center;"><img alt="Stack, heap, queue" src="/files/4617/default.svg" style="height: 270px; width: 294px;"></p>
+<p style="text-align: center;"><img alt="Stack, heap, queue" src="https://developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop/the_javascript_runtime_environment_example.svg"></p>
```

Comment:
Si es posible, evitar los redireccionamientos completos;  esto se puede probar si funciona en entornos locales.

```suggestion
<p style="text-align: center;"><img alt="Stack, heap, queue" src="the_javascript_runtime_environment_example.svg"></p>
```